### PR TITLE
Documents the need for input_pickable in _input_event

### LINKED ISSUE #28306: _input_event not processed by StaticBody2D
**Godot version:**
3.1.stable.flathub

**OS/device including version:**
Linux Mint 19.1

**Issue description:**
I am trying to use _input_event in StaticBody2D, but it doesn't work. Area2D _input_event works properly. I attached small example with two rectangle areas: left is for Area2D, is prints "Area2D pressed" in debug stream when left half of screen is pressed. Right half is covered by StaticBody2D and click on it does not have any effect, but should perform the same way but with little different message.

**Steps to reproduce:**
Download attached project, run it and press on the left part of the screen and on the right part of the screen. You will see only left-related message about Area2D.

**Minimal reproduction project:**
Attached.
[input_event_min_example.zip](https://github.com/godotengine/godot/files/3104182/input_event_min_example.zip)

### PR DESCRIPTION
**Pull request by TheFlamyy**
Closes #28306

### CODE REVIEW

File: doc/classes/CollisionObject2D.xml [hunk 1]
```diff
@@ -19,8 +19,8 @@
 			<argument index="2" name="shape_idx" type="int">
 			</argument>
 			<description>
-				Accepts unhandled [InputEvent]s. [code]shape_idx[/code] is the child index of the clicked [Shape2D]. Connect to the [code]input_event[/code] signal to easily pick up these events.
-			</description>
+				Accepts unhandled [InputEvent]s. Requires [code]input_pickable[/code] to be [code]true[/code]. [code]shape_idx[/code] is the child index of the clicked [Shape2D]. Connect to the [code]input_event[/code] signal to easily pick up these events.
+			</description>28306
```
Comment (YeldhamDev) [inline doc/classes/CollisionObject2D.xml]: This shouldn't be here.

File: doc/classes/CollisionObject2D.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
 			<argument index="2" name="shape_idx" type="int">
 			</argument>
 			<description>
-				Accepts unhandled [InputEvent]s. [code]shape_idx[/code] is the child index of the clicked [Shape2D]. Connect to the [code]input_event[/code] signal to easily pick up these events.
+				Accepts unhandled [InputEvent]s. Requires [code]input_pickable[/code] to be [code]true[/code]. [code]shape_idx[/code] is the child index of the clicked [Shape2D]. Connect to the [code]input_event[/code] signal to easily pick up these events.
```
Comment (akien-mga) [inline doc/classes/CollisionObject2D.xml]: Properties should be referred to with the `[member]` tag to create a hyperlink, here `[member input_pickable]`.

Could you also squash commits together?

Comment (FlamyAT) [reply doc/classes/CollisionObject2D.xml]: I took the opportunity to change all occurrences of `[code]input_pickable[/code]` to `[member input_pickable]`.

I hope I did it right

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**